src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,7 @@
-export const UserInfo = () => (
-  <>Put the user here</>
+export const UserInfo = ({ user }) => (
+  <>
```

Comment:
You can remove fragment `<>` since there are no elements that need to be grouped